### PR TITLE
:memo: Add new block describing how Träwelling handles events

### PR DESCRIPTION
**Pull request by jeyemwey**
As discussed on Discord, using the text from the Google Doc. Additionally, some padding was added between paragraphs on multi-paragraph answers.

### CODE REVIEW

File: resources/lang/en.json [hunk 1]
```diff
@@ -3,6 +3,11 @@
...
     "about.calculation": "The distance is rounded to the nearest 10km and then divided by 10. Afterwards the base points are added.<br />An ICE journey of 143km thus earns you <code>10 + round up(143/10) = 10 + 15 = 25</code>&nbsp;Points, for an S-Bahn journey of 8km you get <code>2 + round up(8/10) = 2 + 1 = 3</code>&nbsp;Points.<br />We want you to check in as soon as possible so that the platform always has an up-to-date status of your journeys. reasonTherefore, you will only receive full points if you check in during your journey or 20min before live departure (scheduled departure + delay). If you check in 1h before or after the travel period, you can still get 1/4 of the points. If you check in earlier or later on a connection, you will receive the full kilometres and hours, but only one pity point.",
     "about.express": "InterCity, EuroCity",
+    "about.events": "What are events?",
+    "about.events.description1": "Träwelling brings people together who use the same public transport. Sometimes those people even ride to the same events, without knowing from another!\n",
```
Comment (MrKrisKrisu) [inline resources/lang/en.json]: is this required?

```suggestion
    "about.events.description1": "Träwelling brings people together who use the same public transport. Sometimes those people even ride to the same events, without knowing from another!",
```

File: resources/views/about.blade.php [hunk 2]
```diff
@@ -88,6 +88,18 @@
                         <p class="lead m-0">{!! __('about.name') !!}</p>
                     </div>
                 </div>
+                <div class="card mb-2">
+                    <div class="card-body">
+                        <h2 class="fs-4 fw-bold text-trwl">
+                            <i class="fa-solid fa-calendar"></i>
+                            {{ __('about.events') }}
+                        </h2>
+                        <p class="lead mb-2">{!! __('about.events.description1') !!}</p>
```
Comment (MrKrisKrisu) [inline resources/views/about.blade.php]:
```suggestion
                        <p class="lead mb-2">{{__('about.events.description1')}}</p>
```

File: resources/views/about.blade.php [hunk 3]
```diff
@@ -88,6 +88,18 @@
...
+                        <p class="lead mb-2">{!! __('about.events.description2', ['link' => route('events.suggest')]) !!}</p>
+                        <p class="lead mb-2">{!! __('about.events.description3') !!}</p>
+                        <p class="lead mb-2">{!! __('about.events.description4') !!}</p>
```
Comment (MrKrisKrisu) [inline resources/views/about.blade.php]:
```suggestion
                        <p class="lead mb-2">{{__('about.events.description4')}}</p>
```

**Review by MrKrisKrisu — CHANGES_REQUESTED** [review]

File: resources/lang/de.json [hunk 4]
```diff
@@ -559,6 +559,11 @@
...
     "to-support": "Zum Support",
     "request-feature": "Funktion wünschen",
+    "about.events": "Was sind Veranstaltungen?",
+    "about.events.description1": "Träwelling bringt Menschen zusammen, die gemeinsam öffentliche Verkehrsmittel nutzen. Manchmal fahren sie sogar zu derselben Veranstaltung, ohne voneinander zu wissen!",
+    "about.events.description2": "Deswegen haben wir das Veranstaltungen-Feature integriert. Jede*r kann <a href=\":link\">über unser Formular</a> eine Veranstaltung für einen bestimmten Zeitraum anlegen und alle Benutzer*innen von Träwelling können ihre Check-ins dann mit dem Event verknüpfen.",
+    "about.events.description3": "Wenn du ein Event anlegen möchtest, beachte bitte, dass Veranstaltungen einen Mehrwert für die gesamte Community von Träwelling haben sollen. Dazu gehören Events aus dem Eisenbahnwesen wie bspw. <a href=\"https://traewelling.de/statuses/event/letzte_fahrt_des_met\">die letzte Fahrt des Metronoms</a>, Events von Hackerspaces (z.B. die <a href=\"https://traewelling.de/statuses/event/gpn20\">GPN</a>), oder aus der LGBTQ-Szene wie die <a href=\"https://traewelling.de/statuses/event/csd_koln_2022\">Cologne Pride</a>.",
```
Comment (MrKrisKrisu) [inline resources/lang/de.json]: oh.. thats what you mean...

met != metronom
met = metropolitan
```suggestion
    "about.events.description3": "Wenn du ein Event anlegen möchtest, beachte bitte, dass Veranstaltungen einen Mehrwert für die gesamte Community von Träwelling haben sollen. Dazu gehören Events aus dem Eisenbahnwesen wie bspw. <a href=\"https://traewelling.de/statuses/event/letzte_fahrt_des_met\">die letzte Fahrt des Metropolitan</a>, Events von Hackerspaces (z.B. die <a href=\"https://traewelling.de/statuses/event/gpn20\">GPN</a>), oder aus der LGBTQ-Szene wie die <a href=\"https://traewelling.de/statuses/event/csd_koln_2022\">Cologne Pride</a>.",
```

**Review by MrKrisKrisu — CHANGES_REQUESTED** [review]

**Comment by jeyemwey** [comment]
Thanks for all the fish, merge if ur ready, @MrKrisKrisu :)

**Review by MrKrisKrisu — APPROVED** [review]

**Pull request merged**